docs/ibc/light-clients/proofs.md

```diff
@@ -1,3 +1,24 @@
 <!--
 order: 7
--->
\ No newline at end of file
+-->
+
+# Existence and Non-Existence Proofs 
+
+`ibc-go` packets are accompanied by proofs as defined in the IBC [ICS23](https://github.com/cosmos/ibc/tree/main/spec/core/ics-023-vector-commitments) specification, which is a general approach for verifying merkle trees.
```

Comment:
I think we should reword this a little. Proofs aren't just used for packet lifecycle so it might be a bit misleading. We use proofs for channel ends, connection ends, client state etc in handshakes too. I wonder should we also link to the ics23 implementation rather than the spec, since the guide is supposed to be more implementation focused. 
The cosmos/ics23 repo also links to the spec.

For the first part, we could say something like:

> IBC uses merkle proofs in order to verify the state of a remote counterparty state machine given a trusted root.

And then follow on to ics23 which provides a generalised approach for proof verification using merkle trees.

Reply:
i do mention the connection handshakes as a place where proofs are used -- should we give a more comprehensive list or simply not list and go with a generalized def, wdyt?

Reply:
I think either is fine. I left a suggestion on latest review for some extra words around existence/non-existence proofs